### ISSUE TITLE
question about continued train only image understanding part.

### ISSUE THREAD

**Issue by switiz** [JiuhaiChen/BLIP3o#21]
Thanks for the great work so far. I want to perform continued training on a model that has already been pre-trained and fine-tuned (SFT), but I’m only interested in image understanding, not generation. So, I believe I need to remove the diffusion part from the modeling loss. How should I go about doing that?

My idea is that if I take a model that has learned both understanding and generation, and fine-tune it on an understanding-only domain, it might perform better than a model that has only learned understanding from the beginning. That’s why I’m asking.

**Comment by JiuhaiChen** [comment]
Hi, thanks for your interest! Our released model freezes the Qwen2.5-VL-7B-Instruct backbone and only trains the image‐generation module, so its understanding capabilities are identical to Qwen2.5-VL-7B-Instruct. 

We also have some ablation checkpoint with joint training of understanding and generation, with SigLIP and llama3-1B-Instruct for image understanding, maybe llama3-1B-Instruct  backbone is too small for your further research?

**Comment by JiuhaiChen** [comment]
Sure, we can prepare the checkpoint, you can also refer to https://github.com/JiuhaiChen/BLIP3o/blob/main/blip3o/train/train.py#L914

You can freeze or unfreeze whatever you want. But the current model is ready for Qwen-VL, we can quickly create a new branch for SigLIP + LLaMA or Qwen, then you could have more flexibility to train the image understanding part.
